data/migrations/20190312170931_messages.js

```diff
@@ -6,6 +6,10 @@ exports.up = function(knex, Promise) {
       .integer("user_id")
       .references("id")
       .inTable("users");
+    tbl
+      .integer("group_id")
```

Comment:
messages shouldn't have both a group_id and a user_id field, since they can be inferred from each other, and this violates third normal form. See here for more information: https://en.wikipedia.org/wiki/Third_normal_form

It's okay since this is just a week project, but it's something to be aware of. In this case, it probably would make sense to remove the user_id and only have a group_id on the message schema.